docs/application/native/guides/multimedia/media-playback.md

```diff
@@ -69,13 +65,13 @@ Tizen enables your application to [play video](#play_video), and it uses its own
 
 The supported video formats include WMV, ASF, MP4, 3GP, AVI, MKV, and OGG. The available formats depend on the target device.
 
-In the case of a video interruption, the state can be READY due to the resource restriction in the system.
+In the case of a video interruption, the state will be one of IDLE, READY or PAUSED due to the resource restriction in the system. You must check the state of the current instance and take appropriate action.
 
 The following figure illustrates what happens when the player gets interrupted by the system.
 
 **Figure: Player states when interrupted by system**
 
-![Player states when interrupted by system](./media/using_videoplayer_interrupted_state.png)
+![Player states when interrupted by system](./media/using_players_interrupted_state.png)
```

Comment:
Suggestion: Request you to please remove line 72 (figure caption) as it is repetitive with line 70